core/controllers/mappings.py

```diff
@@ -533,4 +533,30 @@ def get_row_id(model: Type[BaseModel], lookups: dict[str, Any]) -> str | None:
             ),
         ],
     ),
+    DataMapping(
+        table="Programme Management",
+        model=ents.ProgrammeManagement,
+        column_mapping={
+            "Submission ID": "submission_id",
+            "Programme ID": "programme_id",
+            "Payment Type": "payment_type",
+            "Spend for Reporting Period": "spend_for_reporting_period",
+            "Start_Date": "start_date",
+            "End_Date": "end_date",
+            "Actual/Forecast": "forecast_or_actual",
```

Comment:
Can we standardise this to ```state```, as it appears in Outcomes and Outputs?

Reply:
yeah I was going to ask about that, this was my 50/50 guess 😆